transaction-flow.md

```diff
@@ -55,6 +56,7 @@ Unfortunately not all transactions are successful. The main causes of issues are
 * Transactions are cancelled automatically if they are not funded within the first hour of creation.
 * The recipient details might be wrong (for example the destination bank account number is invalid), or there are other issues blocking the successful payout. In these cases the transaction has to be cancelled.
 * Once the transaction is cancelled, we will refund the money to the sender. Funds from account balances are always refunded automatically to the account balance. Funds from other types of payins might require manual processing however.
+* The `external_id` attached to a transaction already exists in our system.
```

Comment:
this is a validation error and not a transaction failure, so this should be included elsewhere. I would add a full section on the external_id below (where we replaced the details on `sendRef`